releasenotes/notes/1.3/dag-oxide-60b3d7219cb21703.yaml

```diff
@@ -1,25 +1,23 @@
 ---
 features_transpiler:
   - |
-    The implementation of the :class:`.DAGCircuit` has been rewritten in Rust. This rewrite of
-    the Python class should be fully API compatible with the previous Python implementation of
-    the class. While the class was previously implemented using
-    `rustworkx <https://www.rustworkx.org/>`__ and its underlying data graph structure existed
-    in Rust, the implementation of the class and all the data was stored in Python. This new
-    version of :class:`.DAGCircuit` stores a Rust native representation for all its data and
-    is fully implemented in Rust. This new implementation should be more efficient in memory
-    usage as it compresses     the qubit and clbit representation for instructions at rest.
-    It also enables speed up for     transpiler passes as they can fully manipulate a
-    :class:`.DAGCircuit` from Rust.
+    The :class:`.DAGCircuit` has been reimplemented in Rust. This rewrite of the Python class
+    should be fully API compatible with the previous Python implementation. While the class
+    was previously implemented using `rustworkx <https://www.rustworkx.org/>`__, for which
+    the underlying data graph structure exists in Rust, the implementation of the class and
+    all the data was lived in Python. This new version of :class:`.DAGCircuit` stores Rust friendly
```

Comment:
What does "Rust friendly" mean here? The data types are defined in rust instead of python I think the "Rust native representation" in previous version was more descriptive.

Reply:
Yes, I changed the wording here a little because the DAGCircuit still keeps some structures as PyO3-wrapped Python structures so it doesn't have rust native representations for all its data. However, you are right that most of it is, in fact, using Rust native structures, and I should change the wording here.